src/EnergyPlus/OutputReportTabularAnnual.cc

```diff
@@ -0,0 +1,1302 @@
+						for ( unsigned int row = 0; row != m_objectNames.size(); row++ ) { //loop through by row.
+							tableBody( columnRecount - 9 + iBin, row + 1 ) = OutputReportTabular::RealToStr( fldStIt->m_cell[row].m_timeInBin[iBin], fldStIt->m_showDigits );
+						}
+						tableBody( columnRecount - 9 + iBin, rowSumAvg ) = OutputReportTabular::RealToStr( fldStIt->m_timeInBinTotal[iBin], fldStIt->m_showDigits );
+					}
+					columnHead( columnRecount - 10 ) = fldStIt->m_colHead + curAggString + " LESS THAN BIN A";
+					for ( unsigned int row = 0; row != m_objectNames.size(); row++ ) { //loop through by row.
+						tableBody( columnRecount - 10, row + 1 ) = OutputReportTabular::RealToStr( fldStIt->m_cell[row].m_timeBelowBottomBin, fldStIt->m_showDigits );
+					}
+					tableBody( columnRecount - 10, rowSumAvg ) = OutputReportTabular::RealToStr( fldStIt->m_timeBelowBottomBinTotal, fldStIt->m_showDigits );
+					createBinRangeTable = true;
+				}
+				else if (curAgg == AnnualFieldSet::AggregationKind::hoursInTenBinsMinToZero ){
+					// put in the name of the variable for the column
+					if ( fldStIt->m_varAvgSum == isSum ) { // if it is a summed variable
+						curUnits += "/s";
+					}
+					fixUnitsPerSecond( curUnits, curConversionFactor );
+					for ( int iBin = 0; iBin != 10; iBin++ ){
+						char binIndicator = iBin + 65;
+						columnHead( columnRecount - 10 + iBin ) = fldStIt->m_colHead + curAggString + " BIN " + binIndicator;
+						for ( unsigned int row = 0; row != m_objectNames.size(); row++ ) { //loop through by row.
+							tableBody( columnRecount - 10 + iBin, row + 1 ) = OutputReportTabular::RealToStr( fldStIt->m_cell[row].m_timeInBin[iBin], fldStIt->m_showDigits );
+						}
+						tableBody( columnRecount - 10 + iBin, rowSumAvg ) = OutputReportTabular::RealToStr( fldStIt->m_timeInBinTotal[iBin], fldStIt->m_showDigits );
+					}
+					columnHead( columnRecount ) = fldStIt->m_colHead + curAggString + " MORE THAN BIN J";
+					for ( unsigned int row = 0; row != m_objectNames.size(); row++ ) { //loop through by row.
+						tableBody( columnRecount , row + 1 ) = OutputReportTabular::RealToStr( fldStIt->m_cell[row].m_timeAboveTopBin, fldStIt->m_showDigits );
+					}
+					tableBody( columnRecount, rowSumAvg ) = OutputReportTabular::RealToStr( fldStIt->m_timeAboveTopBinTotal, fldStIt->m_showDigits );
+					createBinRangeTable = true;
+				}
+				else if ( curAgg == AnnualFieldSet::AggregationKind::hoursInTenPercentBins || curAgg == AnnualFieldSet::AggregationKind::hoursInTenBinsPlusMinusTwoStdDev ||
+					curAgg == AnnualFieldSet::AggregationKind::hoursInTenBinsPlusMinusThreeStdDev ){
+				}
+			} //fldStIt
+			OutputReportTabular::WriteReportHeaders( m_name, "Entire Facility", isAverage );
+			OutputReportTabular::WriteSubtitle( "Custom Annual Report" );
+			OutputReportTabular::WriteTable( tableBody, rowHead, columnHead, columnWidth, true ); //transpose annual XML tables.
```

Comment:
@JasonGlazer I see that now.

So I *think* what you can do is set each of the streams [\(csv_stream, tab_stream, fix_stream, htm_stream, xml_stream\)](https://github.com/NREL/EnergyPlus/blob/8eeae5e8cc6fdaea95fa5fe188e45c08a8243599/src/EnergyPlus/OutputReportTabular.cc#L199) to use ostringstream, like what I am doing in EnergyPlusFixture for the other streams. So yes, you should add to EnergyPlusFixture just like the other streams. Then as long as you [don't call OpenOutputTabularFile()](https://github.com/NREL/EnergyPlus/blob/8eeae5e8cc6fdaea95fa5fe188e45c08a8243599/src/EnergyPlus/OutputReportTabular.cc#L3032), it should *Just Work :tm:*. Then everything will be output to those ostringstreams and you will be able to compare against them (after you create the new compare functions).